src/routes/api/cryptor-migrations/migration-router.ts

```diff
@@ -0,0 +1,115 @@
+import { Request, Response } from "express";
+import { WhereOptions } from "sequelize";
+import { Installation } from "models/installation";
+
+const MAX_BATCH_SIZE = 100;
+const MAX_TOTAL_LIMIT = 10_000;
+const SAFE_GUARD_LOOP_COUNT = 100;
+
+export const CryptorMigrationInstallationPost = async (req: Request, res: Response): Promise<void> => {
+
+	const startTime = Date.now();
+
+	let batchSize = parseInt(req.query.batchSize as string);
+	let totalLimit = parseInt(req.query.totalLimit as string);
+	const jiraHost = req.query.jiraHost as string;
+
+	req.log = req.log.child({
+		operation: "migrate-installations-shared-secret",
+		batchSize,
+		totalLimit,
+		jiraHost
+	});
+
+	req.log.info(`Received to migration installation sharedSecret for`);
+	if (jiraHost) {
```

Comment:
What's the point of the jiraHost?

Reply:
In case we want to:
1. Only migrate for one particular jiraHost
2. Or we want to re-migrate a previous migrated jiraHost.

Reply:
Well, if you serious don't think we need it, I can remove it completely.

Reply:
fixed.